src/main/com/yetanalytics/squuid/uuid.cljc

```diff
@@ -58,6 +58,16 @@
          (= u1 u2) 0
          (> u1 u2) 1))))
 
+(defn extract-ts-bytes
+  [^UUID uuid]
```

Comment:
Might be useful to have a short docstring here, e.g. "Extracts the first 48 bits of a SQUUID corresponding to a timestamp."